openshift/e2e-tests-openshift.sh

```diff
@@ -19,14 +19,12 @@ source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
 
```

Comment:
this file's changes are identical to what we made for release-v0.9.0 https://github.com/openshift/knative-client/blob/release-v0.9.0/openshift/e2e-tests-openshift.sh

this corresponds to configuring the serverless operator 1.2.0 for client to run tests against